doc-content/enterprise-only/integration/sso-realm-proc.adoc

```diff
@@ -13,8 +13,8 @@ If {CENTRAL} and {KIE_SERVER} are installed on different servers, complete this
 
 * *Product:* Red Hat Single Sign-On
 * *Version:* {RH-SSO_VERSION}
-. Download *Red Hat Single Sign-on {RH-SSO_VERSION_LONG} Server* (`rh-sso-{RH-SSO_VERSION_LONG}.zip`).
-. To install and configure a basic RH-SSO standalone server, follow the instructions in the "Installing and Booting" chapter of the https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/{RH-SSO_VERSION}/html-single/getting_started_guide/#install-boot[_Red Hat Single Sign On Getting Started Guide_]. For advanced settings for production environments, see the https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.3/html-single/server_administration_guide/[_Red Hat Single Sign On Server Administration Guide_].
+. Download *Red Hat Single Sign-on {RH-SSO_VERSION_LONG} Server* (`rh-sso-7.4.1.GA.zip`).
```

Comment:
The actual zip file name is **rh-sso-7.4.1.GA.zip**. So if I use the attribute love there then it will reflect like **rh-sso-7.4.0.zip**. 
Which one is more correct? 
Because I have updated this variable RH-SSO_VERSION_LONG: 7.4.0